server/routes/reports/createReport.ts

```diff
@@ -0,0 +1,226 @@
+// eslint-disable-next-line max-classes-per-file
```

Comment:
There is a lot in this file... :)

Reply:
I think we may want to structure this a bit better somehow.

Mind you this works, it's just a bit chaotic with all these heterogeneous but related things all in a single file 🤔

I wonder if we could introduce a `formWizards` directory alongside routes/views etc...with a directory per view, e.g. `createReport/` and then in this directory we have files with names like `steps.ts`/`fields.ts`/`controllers.ts` and then this file just uses these and it's much smaller.

Thoughts? It doesn't have to happen as part of this PR but I like the idea unless you think it's rubbish. But think about it, this is a relatively simple wizard and the file is already this long, imagine once we add staff/prisoner involvement and what not 😬

Reply:
yup, i agree. this file has grown a fair bit since i started it! mostly because of special date/time handling and validation.

there will be a fair bit of interaction between the stuff in here and the update report version, so it's quite unclear to me what a good structure would look like